src/containers/MoviesContainer/MoviesContainer.js

```diff
@@ -15,11 +15,18 @@ const MoviesContainer = ({ movies }) => {
       overview = {overview}
     />
   })
+  console.log(errorMessage)
```

Comment:
Was this intentionally left here?

Reply:
Nope! But has since been removed, thanks for checking this out.